### ISSUE TITLE
Internal: File uploader event fire twice

### ISSUE THREAD

**Issue by danielholgate** [marklogic-community/pipes#96]
No noticable effect for users, but the CSV file upload component (dropzone) is firing the upload event twice. Investigating.

**Comment by danielholgate** [comment]
Likely need to de-register/move the load event which is attached to fileReader

**Comment by sashamitrovich** [comment]
@danielholgate , is this fixed?

**Comment by danielholgate** [comment]
No, I've tried several things but it remains. Then again, it's not an external problem and doesn't impact anyone. I'm happy close it and remove from the release and just keep track of it internally to fix "one day"

**Comment by sashamitrovich** [comment]
Agreed, moved to Backlog
